modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, `KnativeEventing`, and the `KnativeKafka` custom resource definitions are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have access to the Kafka cluster that produces the Kafka messages you want to import.
```

Comment:
```suggestion
* You have access to a Kafka cluster that produces the Kafka messages you want to import.
```

Reply:
I'd maybe also say events here > messages but I'll leave that to eventing SMEs to decide :)

Reply:
@rhuss @dsimansk WDYT? Kafka messages or Kafka events?

Reply:
IMO `Kafka messages` is the right term here.

Reply:
Yes, the source picks up _Kafka messages_ from Kafka and converts them to _Cloud Events_ (that contain the Kafka message) and propagate them to a _sink_ (so, the sink always receives cloud events)

Reply:
@rhuss Thanks for the explanation on messages vs events